### ISSUE TITLE
Possible issue w/ PHP < 5.4 in JP 4.8

### ISSUE THREAD

**Issue by dbspringer** [Automattic/jetpack#6931]
Capturing conversation w/ @jeherve. Updated to Jetpack 4.8 on a GoDaddy hosted, fully updated version of WordPress (4.7.3) and experienced white screen of death.

**Actions Taken**
* Deleted Jetpack DIR via FTP to restore site
* Reinstalled Jetpack plugin and got caught in "please sync" loop

Pulled this error out of the logs:
```PHP Parse error:  syntax error, unexpected '[' in /home/content/51/9498351/html/wp-content/plugins/jetpack/modules/sitemaps/sitemap-state.php on line 87```

PHP version: 5.3.24

**Comment by dbspringer** [comment]
Digging into the error log some more, pulled this out just now when trying to delete plugin via `/wp-admin/plugins.php`:

```
[05-Apr-2017 19:22:10 UTC] PHP Warning:  require_once(/home/content/51/9498351/html/wp-content/plugins/jetpack/sync/class.jetpack-sync-sender.php) [<a href='function.require-once'>function.require-once</a>]: failed to open stream: No such file or directory in /home/content/51/9498351/html/wp-content/plugins/jetpack/sync/class.jetpack-sync-actions.php on line 277
[05-Apr-2017 19:22:10 UTC] PHP Fatal error:  require_once() [<a href='function.require'>function.require</a>]: Failed opening required '/home/content/51/9498351/html/wp-content/plugins/jetpack/sync/class.jetpack-sync-sender.php' (include_path='.:/usr/local/php5_3/lib/php') in /home/content/51/9498351/html/wp-content/plugins/jetpack/sync/class.jetpack-sync-actions.php on line 277
[05-Apr-2017 19:22:12 UTC] PHP Warning:  stat(): stat failed for /home/content/51/9498351/html/wp-content/plugins/jetpack/CODE-OF-CONDUCT.md in /home/content/51/9498351/html/wp-content/plugins/vaultpress/vaultpress.php(1374) : eval()'d code on line 1
```

**Comment by dereksmart** [comment]
This should be fixed by the latest 4.8.1 release.  Specifically this patch #6920
